datatypes/src/raster/raster_properties.rs

```diff
@@ -21,10 +21,51 @@ impl RasterProperties {
         self.properties_map.keys().map(|m| m.domain.as_ref())
     }
 
+    pub fn set_scale(&mut self, scale: f64) {
+        self.scale = Some(scale);
```

Comment:
btw is this the value this was scaled with or is this the value we have to scale the pixels with to get measurements?

Reply:
it is the value the data was scaled with

Reply:
maybe add some docs somewhere or refer to the Gdal docs?

Reply:
[ad30cb7](https://github.com/geo-engine/geoengine/pull/530/commits/ad30cb7840fa6e361f40e368a88b5148498b7d56)